index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ali Ahmad Karimi Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+    
+</head>
+
+<body>
+    <section>
+        <nav class="toolbar">
+            <h3 class="logo">Ali Ahmad Karimi</h3>
+            <img src="images/Icon-Menu.png" alt="Menu Icon">
+        </nav>
```

Comment:
Please note the following: 😁😁

- [ ] The toolbar section, should be wrapped in a `header` tag and should be separated from the other section, kindly change that.
- [ ] Only the Menu Icon should be inside a `nav` tag, kindly change that as well.
- [ ] The logo `<h3 class="logo">Ali Ahmad Karimi</h3>` should be inside an `a` tag, kindly do that.
- [ ] The weight of the logo text isn't the same one that is mentioned in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) (please check my comment about the way you imported the font that should fix this issue).